jgt_tools/utils.py

```diff
@@ -19,16 +21,16 @@ def execute_command_list(commands_to_run, verbose=True):
         subprocess.run(shlex.split(command), check=True)
 
 
-_DEFAULT_CONFIGS = {
-    "env_setup_commands": [
-        "pip install --upgrade pip<19",
-        "poetry install",
-        "pre-commit install",
-    ],
-    "self_check_commands": ["pre-commit run -a"],
-    "run_tests_commands": [f"{sys.executable} -m pytest"],
-    "doc_build_types": ["api"],
-}
+_DEFAULT_CONFIGS: defaultdict = defaultdict(list)
```

Comment:
Since we aren't doing `mypy` in this project (yet?), should this just be `_DEFAULT_CONFIGS = defaultdict(list)`?

Reply:
I have intentions to add annotations shortly, but we already have `mypy` as a `pre-commit` checker, and it was barking at me about this one, so I put this in place now.